js/path.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+
+export function pathBackwards(path: string): string {
+  path.trim();
```

Comment:
`trim` returns a new value, it should be `path = path.trim()` (same issue in the other function).
There may also be edge cases around trimming, it would benefit from having test cases checking it.

Reply:
Thanks,  fixed it.